cnf-tests/mirror/mirror.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+)
+
+type image struct {
+	Name     string `json:"name"`
+	Registry string `json:"registry"`
+	Version  string `json:"version"`
+}
+
+func main() {
+	imagesFile := flag.String("images", "/usr/local/etc/cnf/images.json", "the json file containing the images")
+	targetRegistry := flag.String("registry", "", "the target registry we want to mirror to")
+
+	flag.Parse()
+
+	if *imagesFile == "" || *targetRegistry == "" {
+		flag.Usage()
+		log.Fatal("Missing mandatory fields")
+	}
+
+	bytes, err := ioutil.ReadFile(*imagesFile)
```

Comment:
Can you validate the file exist before you read it?

Reply:
Not sure I see the need. If the file does not exist, readfile will return an error and the command will fail anyway saying "I can't read it". The user will then check the file and see what's wrong with it.
I think it's not worth adding other 4 lines of code.